docker-compose.yml

```diff
@@ -35,6 +35,7 @@ nginx:
     - "80:80"
   volumes:
     - ./static:/srv/static
```

Comment:
Yes because otherwise tree changes aren't there without running the make update_assets command to copy things over. I did try it.